Various cleanups to the Print layout lesson (#5701)
qgis/QGIS-Documentation: docs/training_manual/map_composer/map_composer.rst

   
#. You can safely use the default values now and click
   :guilabel:`Save`.
   
   QGIS will proceed to the map export and push a message
   on top of the print layout dialog as soon as it finishes.

> #. Click the hyperlink in the message to open in your system's file
>    manager the folder in which the PDF has been saved

Reviewer: ```suggestion
#. Click the hyperlink in the message to open the folder in which
   the PDF has been saved in your system's file  manager
```

> #. Open it and watch how it looks like your layout in QGIS.

Reviewer: ```suggestion
#. Open it to see how your QGIS layout looks.
```

   Congratulations on your first completed QGIS map project!

#. Back to the QGIS window, save your project file
#. Close the layout window and the project.

|IC|
----------------------------------------------------------------------

> You've learned to create a basic static map layout, let's go a step

Reviewer: ```suggestion
Now you know how to create a basic static map layout. Now we can go a step
```
